notebooks/04/eda_disk_to_mem.md

```diff
@@ -0,0 +1,431 @@
+
+
+```python
+# HIDDEN
+import warnings
+# Ignore numpy dtype warnings. These warnings are caused by an interaction
+# between numpy and Cython and can be safely ignored.
+# Reference: https://stackoverflow.com/a/40846742
+warnings.filterwarnings("ignore", message="numpy.dtype size changed")
+warnings.filterwarnings("ignore", message="numpy.ufunc size changed")
+
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+%matplotlib inline
+import ipywidgets as widgets
+from ipywidgets import interact, interactive, fixed, interact_manual
+import nbinteract as nbi
+
+sns.set()
+sns.set_context('talk')
+np.set_printoptions(threshold=20, precision=2, suppress=True)
+pd.options.display.max_rows = 7
+pd.options.display.max_columns = 8
+pd.set_option('precision', 2)
+# This option stops scientific notation for pandas
+# pd.set_option('display.float_format', '{:.2f}'.format)
+```
+
+## From Disk to Memory
```

Comment:
Let's move Structure first, then Disk to Memory.

Let's move this section at the start of ch3.